### PR TITLE
Add 0.16.0.0 for dustinface

### PR DESCRIPTION
**Pull request by xdustinface**

### CODE REVIEW

**Review by UdjinM6 — COMMENTED** [review]
Looks good so far

File: 0.16.0.0-osx-signed/dustinface/dash-dmg-signer-build.assert [hunk 1]
```diff
@@ -0,0 +1,619 @@
+--- !omap
+- out_manifest: |
+    50d3b91904111d1fd9b2bccbbb7ecee30e60a23ce98ce234442e61325bfa3784  dashcore-osx-signed.dmg
```
Comment (thephez) [inline 0.16.0.0-osx-signed/dustinface/dash-dmg-signer-build.assert:3]: This looks different than mine :thinking: They are supposed to match, right?
https://github.com/dashpay/gitian.sigs/pull/92/files#diff-9aa2791b84c32e00877f72b1095bd595R2

**Review by UdjinM6 — DISMISSED** [review]
ACK

**Review by PastaPastaPasta — DISMISSED** [review]
ACK, gitian-build.py --verify is happy :+1:

**Review by UdjinM6 — APPROVED** [review]
ACK

**Review by PastaPastaPasta — APPROVED** [review]
ACK

**Pull request merged**